fuel-coroutines/src/test/kotlin/com/github/kittinunf/fuel/coroutines/ObjectTest.kt

```diff
@@ -67,9 +69,9 @@ class ObjectTest : MockHttpTestCase() {
     fun awaitObjectResponse() = runBlocking {
         try {
             val (request, response, data) = randomUuid().awaitObjectResponse(UUIDResponseDeserializer)
-            assertThat("Expected request to be not null", request, notNullValue())
-            assertThat("Expected response to be not null", response, notNullValue())
-            assertThat("Expected data to be not null", data, notNullValue())
```

Comment:
The reason these messages are here is that in some context such as CI you only get `expected not null , actual null`. These messages change the output to be `expected xxx to be not null`.

Reply:
I think we can pin down to the line that throws Error right? I find it redundant ...

Anyway if you want this to stay, I will revert it back.

Reply:
> I think we can pin down to the line that throws Error right? I find it a bit redundant 🤔
Not on CI! Or am I wrong?